content/en/about/feature-stages/index.md

```diff
@@ -16,17 +16,29 @@ within the project, not to the project as a whole. Here is a high level descript
+| Deprecation Policy    | None, feature can be removed at any time                                                                                                                                                                                                                                                                                                                                                                                                                                   | None, feature can be removed at any time                                                                                                                                                                                                                                                                                                                                                                                                           | Weak, feature can be removed with 3 months of advanced notice                                                                                                                                                                                                                                                                                                                                                                                                                               | Strong, feature can be removed with 1 year of advanced notice, but there will usually be a supported upgrade path to a replacement feature |
```

Comment:
```suggestion
| Deprecation Policy    | None, feature can be removed at any time.                                                                                                                                                                                                                                                                                                                                                                                                                                   | None, feature can be removed at any time.                                                                                                                                                                                                                                                                                                                                                                                                           | Weak, feature can be removed with 3 months of advanced notice.                                                                                                                                                                                                                                                                                                                                                                                                                               | Strong, feature can be removed with 1 year of advanced notice, but there will usually be a supported upgrade path to a replacement feature. |
```

Reply:
There is a lot of inconsistenty with periods.  https://deploy-preview-8903--preliminary-istio.netlify.app/latest/about/feature-stages/ for current deployment. Sometimes it missing in some columns of a row. Sometime for a whole column.